lib/routes.js

```diff
@@ -24,6 +24,33 @@ module.exports = function(self, options) {
     });
   });
 
+  self.route('post', 'revert', function(req, res) {
+    var id = self.apos.launder.id(req.body.id);
+    return self.revert(req, id, function(err, result) {
+      if (err) {
+        return res.send({
+          status: (typeof (err) === 'string') ? err : 'error'
+        });
+      }
+
+      // On success, grab current version of reverted doc and include url for client-side redirect
+      return self.apos.docs.find(req, { _id: result.id }).toArray(function(err, docs) {
```

Comment:
Pretty sure this is going to be `result._id`, yes? Not `result.id`.

Please add a browser functional test of the redirect behavior we're ultimately trying to achieve here so this gets completely debugged.

Reply:
On second thought, if `self.apos.docs.update` returns the updated draft, we shouldn't need to reload it from the db.